src/main/kotlin/com/re/back/dtos/response/AuthResponseDto.kt

```diff
@@ -0,0 +1,14 @@
+package com.re.back.dtos.response
+
+import com.re.back.enums.UserRole
+import java.time.LocalDateTime
+
+data class AuthResponseDto(
+    val id : Int?,
+    val userName : String,
+    val email : String,
+    val role : UserRole,
+    val bio : String?,
+    val token : String,
+    val expiresOn : LocalDateTime
```

Comment:
Curious about why you return expires on?

Reply:
That for frontend/mobile developers use to know when the token expired. Later we can add `/refresh-token` endpoint used by them.
We use this approach most of the time